poppy/poppy_core.py

```diff
@@ -2273,8 +2273,10 @@ def get_phasor(self,wave):
                 lx,ly=resampled_amplitude.shape
                 #crop down to match size of wavefront:
                 lx_w,ly_w = wave.amplitude.shape
-                border_x = np.abs(np.floor((lx-lx_w)/2))
-                border_y = np.abs(np.floor((ly-ly_w)/2))
+
+                # explicit cast to int fixes index errors in Numpy >= 1.12.0
+                border_x = int(np.abs(np.floor((lx-lx_w)/2)))
```

Comment:
I'm on the fence about whether we need explanatory comments on each instance of this fix. It makes it clearer now what the logic is for the changes, but thinking about this from the perspective of code maintenance 2-5 years from now I'm not sure it enhances code readability to keep drawing attention to the index errors then. @josePhoenix, @douglase opinions on what's best for signal to noise here?